packages/xo-web/src/xo-app/tasks/index.js

```diff
@@ -220,6 +220,14 @@ const INDIVIDUAL_ACTIONS = [
   },
 ]
 
+const XO_TASKS_INDIVIDUAL_ACTIONS = [
+  {
+    handler: task => window.open(task.href),
```

Comment:
There is always the issue with absolute/relative path and reverse proxies but I don't see any ideal solution and the problem is very small, therefore let's ignore this for now.